examples/flux-source/app/index.ts

```diff
@@ -0,0 +1,51 @@
+import * as k8s from "@pulumi/kubernetes";
+
+
+// A git repository source for our Pulumi program
+const gitRepo = new k8s.apiextensions.CustomResource("pko-dev", {
+    apiVersion: 'source.toolkit.fluxcd.io/v1beta2',
+    kind: 'GitRepository',
+    metadata: {
+        'namespace': 'default',
+    },
+    spec: {
+        interval: '5m0s',
+        url: 'https://github.com/squaremo/pko-dev',
+        ref: { branch: 'main' },
+    },
+});
+
+// A secret with the Pulumi access token, taken from the environment.
+const tokenSecret = new k8s.core.v1.Secret("pulumi-token", {
+    stringData: {
+        'PULUMI_ACCESS_TOKEN': process.env['PULUMI_ACCESS_TOKEN'] || 'token',
+    },
```

Comment:
Minor - but seems `”token”` will never be valid. Should this be pulled from a Pulumi config secret?